<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fractions</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout5">
     <item>
      <widget class="QPushButton" name="submit_button">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_button">
       <property name="text">
        <string>Reset All</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontaLayout3">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Arithmetic</string>
       </property>
       <widget class="QComboBox" name="arith_op_combo_box">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout2">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Fraction: 2</string>
       </property>
       <widget class="QLineEdit" name="numerator_line_edit_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>50</width>
          <height>20</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Numerator#1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="div_sign_label_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>10</y>
          <width>4</width>
          <height>39</height>
         </rect>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="denominator_line_edit_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>First Name</string>
        </property>
       </widget>
       <widget class="QPushButton" name="inverse_button_2">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>20</y>
          <width>51</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Inverse</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="text_edit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout1">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Fraction: 1</string>
       </property>
       <widget class="QLineEdit" name="numerator_line_edit_1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>50</width>
          <height>20</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Numerator#1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="div_sign_label_1">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>10</y>
          <width>4</width>
          <height>39</height>
         </rect>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="denominator_line_edit_1">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>First Name</string>
        </property>
       </widget>
       <widget class="QPushButton" name="inverse_button_1">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>20</y>
          <width>51</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Inverse</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout4">
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Logical</string>
       </property>
       <widget class="QComboBox" name="log_op_combo_box">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="pic_label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>403</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>